docs/conceptual/node/tx-pool.md

```diff
@@ -2,12 +2,80 @@
 title: Transaction Pool
 ---
 
-* What is the Transaction Pool?
 
-* What is priority?
+The transaction pool contains all transactions (signed and unsigned) broadcasted to the network that have been *received* and *validated* by the local node. The pool is responsible for:
 
-* What is dependency?
 
-* Graph VS Queue
+1) Checking if transactions are valid. `validity` of the transaction is not hard wired to the transaction pool, but is defined by the runtime through the `validate_transaction` function ([code](https://github.com/paritytech/substrate/blob/20a9b15cdbed4bf962a4447e8bfb812f766f2fbc/frame/executive/src/lib.rs#L304)). Example validity checks are:
 
-* What is the lifecycle of a transaction?
+* Transaction Index (nonce) correctness.
+* The account has enough funds to pay for the associated fees.
+* The signature is valid.
+
+2) If the transaction is valid, sorting the transactions into 2 transaction queues:
```

Comment:
The rendered version of this is borked. I would remove the `1)` and `2)` and just make two bold-faced lines with paragraphs/bullets underneath them. Like:
```
**Validity**

The transaction pool checks for transaction validity to... <why>. For example, it checks that:

**Sorting**

The transaction queue sorts transactions into two groups, the "ready queue" and the "future queue".
```